lib/internal/process.js

```diff
@@ -65,6 +65,22 @@ function setupConfig(_source) {
     if (value === 'false') return false;
     return value;
   });
+  // Intl.v8BreakIterator() without data crashes with a fatal error, so throw instead.
+  if (global.hasOwnProperty('Intl') &&
+      Intl.hasOwnProperty('v8BreakIterator') &&
+      process.hasOwnProperty('config') &&
+      process.config.hasOwnProperty('variables') &&
+      process.config.variables.icu_small &&
+      !process.icu_data_dir) {
```

Comment:
OK - what's the recommendation?

Probably worst case is the userland code causes node to crash because it can't detect that it was about to crash. Is there a workaround?